visualize-admin/visualization-tool
# feat: Use ordinal measures with maps

Closes #734.
Fixes #781.

This PR introduces the usage of ordinal measures in maps (both in area and symbol layers). Additionally, we now fetch optional colors assigned to dimension values and use them as default, initial color values in color mappings. It also adds title for non-hierarchical map legend.

### How to test
1. Go to `/en/browse/dataset/https%3A%2F%2Fenvironment.ld.admin.ch%2Ffoen%2Ftest_ordinal_507%2F7?dataSource=Int`
2. Select map chart type.
3. Select area layer.
4. See that `ord1` (ordinal measure) is selected and it's possible to select other ordinal measures.
5. See that the colors are custom-defined (green-red).
6. Select symbol layer.
7. Enable the layer.
8. See that in this **_extreme_** example we don't have any numerical measure so the radius of the symbols is fixed and shared across all symbols.
9. Change color mapping and see that it's possible to choose ordinal measure to color the symbols.
